run.20230215.NGD_v3atm.F2010.P10.sh

```diff
@@ -0,0 +1,657 @@
+         'A:so4_a2:N:so4_c2:sulfate:\$DIN_LOC_ROOT/atm/cam/physprops/sulfate_rrtmg_c080918.nc:+',
+         'A:soa_a2:N:soa_c2:s-organic:\$DIN_LOC_ROOT/atm/cam/physprops/ocphi_rrtmg_c100508.nc:+',
+         'A:ncl_a2:N:ncl_c2:seasalt:\$DIN_LOC_ROOT/atm/cam/physprops/ssam_rrtmg_c100508.nc:+',
+         'A:mom_a2:N:mom_c2:m-organic:\$DIN_LOC_ROOT/atm/cam/physprops/poly_rrtmg_c130816.nc:',
+         'mam5_mode3:coarse:=', 'A:num_a3:N:num_c3:num_mr:+',
+         'A:dst_a3:N:dst_c3:dust:\$DIN_LOC_ROOT/atm/cam/physprops/dust_aeronet_rrtmg_c141106.nc:+',
+         'A:ncl_a3:N:ncl_c3:seasalt:\$DIN_LOC_ROOT/atm/cam/physprops/ssam_rrtmg_c100508.nc:+',
+         'A:so4_a3:N:so4_c3:sulfate:\$DIN_LOC_ROOT/atm/cam/physprops/sulfate_rrtmg_c080918.nc:+',
+         'A:bc_a3:N:bc_c3:black-c:\$DIN_LOC_ROOT/atm/cam/physprops/bcpho_rrtmg_c100508.nc:+',
+         'A:pom_a3:N:pom_c3:p-organic:\$DIN_LOC_ROOT/atm/cam/physprops/ocpho_rrtmg_c130709.nc:+',
+         'A:soa_a3:N:soa_c3:s-organic:\$DIN_LOC_ROOT/atm/cam/physprops/ocphi_rrtmg_c100508.nc:+',
+         'A:mom_a3:N:mom_c3:m-organic:\$DIN_LOC_ROOT/atm/cam/physprops/poly_rrtmg_c130816.nc:',
+         'mam5_mode4:primary_carbon:=', 'A:num_a4:N:num_c4:num_mr:+',
+         'A:pom_a4:N:pom_c4:p-organic:\$DIN_LOC_ROOT/atm/cam/physprops/ocpho_rrtmg_c130709.nc:+',
+         'A:bc_a4:N:bc_c4:black-c:\$DIN_LOC_ROOT/atm/cam/physprops/bcpho_rrtmg_c100508.nc:+',
+         'A:mom_a4:N:mom_c4:m-organic:\$DIN_LOC_ROOT/atm/cam/physprops/poly_rrtmg_c130816.nc',
+         'mam5_mode5:strat_coarse:=', 'A:num_a5:N:num_c5:num_mr:+',
+         'A:so4_a5:N:so4_c5:sulfate:\$DIN_LOC_ROOT/atm/cam/physprops/sulfate_rrtmg_c080918.nc'
+
+ rad_climate            = 'A:H2OLNZ:H2O', 'N:O2:O2', 'N:CO2:CO2',
+         'A:O3:O3', 'A:N2OLNZ:N2O', 'A:CH4LNZ:CH4',
+         'N:CFC11:CFC11', 'N:CFC12:CFC12', 
+         'M:mam5_mode1:\$DIN_LOC_ROOT/atm/cam/physprops/mam4_mode1_rrtmg_aeronetdust_c141106.nc',
+         'M:mam5_mode2:\$DIN_LOC_ROOT/atm/cam/physprops/mam4_mode2_rrtmg_c130628.nc', 
+         'M:mam5_mode3:\$DIN_LOC_ROOT/atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_c141106.nc',
+         'M:mam5_mode4:\$DIN_LOC_ROOT/atm/cam/physprops/mam4_mode4_rrtmg_c130628.nc',
+         'M:mam5_mode5:\$DIN_LOC_ROOT/atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219_ke.nc'
+! rad_diag_1 = 'A:H2OLNZ:H2O','N:O2:O2','N:CO2:CO2','A:O3:O3','A:N2OLNZ:N2O','A:CH4LNZ:CH4','N:CFC11:CFC11','N:CFC12:CFC12'
+
+
+ cflx_cpl_opt = 2
+
+ soil_erod_file         = '\$DIN_LOC_ROOT/atm/cam/dst/dst_1.9x2.5_c090203.nc'
+
+ megan_factors_file = '\$DIN_LOC_ROOT/atm/cam/chem/trop_mozart/emis/megan21_emis_factors_c20130304.nc'
+ megan_specifier = 'C10H16 = myrcene + sabinene + limonene + carene_3 + ocimene_t_b + pinene_b + pinene_a + 2met_styrene + cymene_p + cymene_o + phellandrene_a + thujene_a + terpinene_a + terpinene_g + terpinolene + phellandrene_b + camphene + bornene + fenchene_a + ocimene_al + ocimene_c_b'
+
```

Comment:
we will not use megan in the any future runs. We need to remove the two lines (megan_factors_file and megan_specifier). Also need to change amip run script.